README.md

```diff
@@ -59,54 +69,114 @@ Some components or modules in forte may require some [extra requirements](https:
-* `pip install forte[extractor]`: Install packages required for extrator-based training system, [extractor](https://github.com/asyml/forte/blob/master/forte/data/extractors), [train_preprocessor](https://github.com/asyml/forte/tree/master/forte/train_preprocessor.py) and [tagging trainer](https://github.com/asyml/forte/tree/master/examples/tagging/tagging_trainer.py)
-
+* `pip install forte[extractor]`: Install packages required for extractor-based training system, [extractor](https://github.com/asyml/forte/blob/master/forte/data/extractors), [train_preprocessor](https://github.com/asyml/forte/tree/master/forte/train_preprocessor.py) and [tagging trainer](https://github.com/asyml/forte/tree/master/examples/tagging/tagging_trainer.py)
 
 
-## Getting Started
-
-* [Examples](./examples)
-* [Documentation](https://asyml-forte.readthedocs.io/)
-* Currently we are working on some
-  interesting [tutorials](https://github.com/asyml/forte/wiki)
-
 
-With Forte, it is extremely simple to build an integrated system that can search
-documents, analyze, extract information and generate language all in one place.
-This allows developers to fully utilize the strength of individual module,
-combine the results from each step, and enables the system to make fully
-informed decision at the end of the pipeline.
-
-Forte not only makes it easy to integrate with arbitrary 3rd party tools (Check
-out these [examples](./examples)!), but also brings technology to you by
-offering a miscellaneous collection of deep learning modules via Texar, and a
-convenient model-data interface for casting tasks to models.
-
-## Library Example
-
-A simple code example that runs Named Entity Recognizer from Spacy (required
-installing forte spacy wrapper)
+## Quick Start Guide
+Here we provide an example for user to get started with Forte easier. Consider a case we want to get words in the sentence and also extract entities from the sentence. We can write two processors for the two separate tasks.
 
+First, we imports all required libraries.
 ```python
 from forte import Pipeline
+from forte.processors.base import PackProcessor
+from forte.data.data_pack import DataPack
+from dataclasses import dataclass
+from forte.data.ontology.top import Generics
+from typing import Optional, Dict, Any
 from forte.data.readers import TerminalReader
 from fortex.spacy import SpacyProcessor
+import re
+```
+In Forte, we use data entry with different entry types to represent data.
+The following class is an example of data entry storing word texts in the class variable `value`.
+New User doesn't need to pay too much attention about it and can just use it as it is.
 
-for pack in Pipeline().set_reader(
-        TerminalReader()
-).add(
-    SpacyProcessor(), {"processors": ["sentence", "ner"]}
-).initialize().process_dataset():
+```python
+@dataclass
+class Word(Generics):
```

Comment:
Let's just don't add the new `Word` class here. It is too deep for "Quick Start", just use the pre-built classes from base_ontology.